utils/Config.cpp

```diff
@@ -77,3 +121,28 @@ std::string const Config::defaultErrorPageBody(int code) {
 
   return ss.str();
 }
+
+// path에 포함된 확장자에 맞는 MIME 타입 반환
+// - 알 수 없는 확장자의 경우 기본 설정된 MIME 타입 반환
+std::string const Config::findMimeType(const std::string& path) {
+  // 마지막 '.' 위치를 찾음
+  size_t dotPos = path.find_last_of('.');
+  std::string extension;
+
+  if (dotPos != std::string::npos) {
+    // 확장자 추출 (마지막 '.' 이후의 모든 문자)
+    extension = path.substr(dotPos);
+  } else {
+    // 확장자가 없는 경우 기본 MIME 타입 반환
+    return "application/octet-stream";
+  }
+
+  // 확장자에 맞는 MIME 타입 반환
+  std::map<std::string, std::string>::const_iterator it =
+      mimeTypes.find(extension);
+  if (it != mimeTypes.end()) {
+    return it->second;
+  } else {
+    return "application/octet-stream";
+  }
```

Comment:
여기서 else 문을 사용하지 않고 바로 return "application/octet-stream"으로 작성해도 좋을 것 같아요!

```suggestion
  if (it != mimeTypes.end()) {
    return it->second;
  } 
  return "application/octet-stream";
```